invokeai/app/invocations/image_to_latents.py

```diff
@@ -130,6 +143,11 @@ def invoke(self, context: InvocationContext) -> LatentsOutput:
         assert isinstance(vae_info.model, (AutoencoderKL, AutoencoderTiny))
 
         image_tensor = image_resized_to_grid_as_tensor(image.convert("RGB"))
+
+        if self.color_compensation != "None":
```

Comment:
What happens when  color compensation is set to "SDXL" but an SD1.5 VAE is applied? If possible, you should check that the VAE has base type SDXL before applying compensation.